tests/rkt_app_sandbox_test.go

```diff
@@ -75,6 +75,7 @@ func TestAppSandboxSmoke(t *testing.T) {
 	cmd = strings.Fields(fmt.Sprintf("%s app add --debug %s %s --name=%s", ctx.Cmd(), podUUID, imageName, appName))
 	addCmd := exec.Command(cmd[0], cmd[1:]...)
 	addCmd.Env = append(addCmd.Env, "RKT_EXPERIMENT_APP=true")
+	fmt.Printf("Running command: %v\n", cmd)
```

Comment:
ah, I left it there, because this output is unbuffered, but I'll change it back to `t.Logf`.